addons/l10n_it/data/account.account.template.csv

```diff
@@ -1,185 +1,185 @@
-1101,1101,Costi di impianto,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
+1101,1101,Planting costs,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
 1106,1106,Software,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1108,1108,Avviamento,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1111,1111,Fondo ammortamento costi di impianto,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1116,1116,Fondo ammortamento software,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1118,1118,Fondo ammortamento avviamento,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1201,1201,Fabbricati,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1202,1202,Impianti e macchinari,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1204,1204,Attrezzature commerciali,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1205,1205,Macchine d'ufficio,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1206,1206,Arredamento,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1207,1207,Automezzi,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1208,1208,Imballaggi durevoli,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1211,1211,Fondo ammortamento fabbricati,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1212,1212,Fondo ammortamento impianti e macchinari,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1214,1214,Fondo ammortamento attrezzature commerciali,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1215,1215,Fondo ammortamento macchine d'ufficio,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1216,1216,Fondo ammortamento arredamento,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1217,1217,Fondo ammortamento automezzi,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1218,1218,Fondo ammortamento imballaggi durevoli,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1220,1220,Fornitori immobilizzazioni c/acconti,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1301,1301,Mutui attivi,asset_fixed,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1401,1401,Materie di consumo,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
-1404,1404,Merci,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
-1410,1410,Fornitori c/acconti,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
```

Comment:
Could you please translate this file again after removing the abbreviations like : c/, v/, n/...